geode-core/src/test/java/org/apache/geode/management/internal/cli/help/HelperIntegrationTest.java

```diff
@@ -34,7 +36,7 @@
   public static void beforeClass() {
     helper = new Helper();
     // use GfshHelpCommand for testing
-    Method[] methods = GfshHelpCommands.class.getMethods();
+    Method[] methods = GfshHelpCommand.class.getMethods();
```

Comment:
With the original multiple-command class being split, this test now only checks one of the two commands. Add a second test for GfshHintCommand. I would add it to this test class, refactoring the beforeClass functionality into separate imnplementations in each test.